ruby-lsp-rails.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.metadata["source_code_uri"] = spec.homepage
   spec.metadata["changelog_uri"] = "#{spec.homepage}/releases"
 
-  spec.required_ruby_version = ">= 3.0.0"
+  spec.required_ruby_version = ">= 3.1.0"
```

Comment:
We didn't drop support for Ruby 3.0 in the Ruby LSP, we just removed it from CI because `rbi` dropped support for it and bundle install fails.

I know 3.0 is already EOL, but can we please wait a little longer and avoid another situation like Ruby 2.7?

There are no features we want to use that depend on Ruby > 3.0 anyway.

Reply:
Restored.